### PR TITLE
kde-frameworks: update to 5.48.0

### PR DESCRIPTION
**Pull request by Johnnynator**

### CODE REVIEW

**Comment by yopito** [comment]
@Johnnynator : nice work !  
There's some missing library according to https://api.kde.org/frameworks/index.html:
- [ ]  KUserFeedback
- [ ] Qqc2-desktop-style 
- [ ] Purpose 

do you plan to package these in this PR ? 
I don't know which are (optional) dep of other, but at least, "purpose" is an optional dep for kdevelop.

**Comment by Johnnynator** [comment]
I planned to add at least Purpose, can look at the other 2 two too.
Will also have to take a look at cross compiling (at least `ktexteditor` was failing to compile)
@Chocimier Do you had any plans regarding Cross Compilation for this Update, kauth is making problems as always.

**Comment by Chocimier** [comment]
No, if hostmakedepends from kate don't help, we need to wait for upstream fix.

**Comment by Johnnynator** [comment]
@Chocimier Thanks for your feedback. I will set affected packages to nocross for now.

**Comment by Johnnynator** [comment]
- [ ] qqc2-style-integration probably needs to be added to plasma-integration as a dep to take effect (not important for this PR)
 KUserFeedback
- [ ] KUserFeedback, Telemetry and User Feedback. Will not add it for now.
- [ ] purpose: Will add it on a later occasion (and separate PR)

File: srcpkgs/attica-qt5/template [hunk 1]
```diff
@@ -12,7 +12,7 @@ maintainer="Juan RP <xtraeme@gmail.com>"
 license="LGPL-2.1"
```
Comment (maxice8) [inline srcpkgs/attica-qt5/template]: spdx

File: srcpkgs/bluez-qt5/template [hunk 2]
```diff
@@ -13,7 +13,7 @@ maintainer="Denis Revin <denis.revin@gmail.com>"
 license="LGPL-2"
```
Comment (maxice8) [inline srcpkgs/bluez-qt5/template]: spdx

File: srcpkgs/breeze-icons/template [hunk 3]
```diff
@@ -10,7 +10,7 @@ maintainer="Juan RP <xtraeme@voidlinux.eu>"
 license="LGPL-3"
```
Comment (maxice8) [inline srcpkgs/breeze-icons/template]: spdx

File: srcpkgs/frameworkintegration/template [hunk 4]
```diff
@@ -11,7 +11,7 @@ maintainer="Denis Revin <denis.revin@gmail.com>"
 license="LGPL-2"
```
Comment (maxice8) [inline srcpkgs/frameworkintegration/template]: spdx

File: srcpkgs/kactivities5-stats/template [hunk 5]
```diff
@@ -12,7 +12,7 @@ maintainer="Denis Revin <denis.revin@gmail.com>"
 license="LGPL-2"
```
Comment (maxice8) [inline srcpkgs/kactivities5-stats/template]: spdx

File: srcpkgs/kapidox/template [hunk 6]
```diff
@@ -13,7 +13,7 @@ maintainer="Denis Revin <denis.revin@gmail.com>"
 license="2-clause-BSD"
```
Comment (maxice8) [inline srcpkgs/kapidox/template]: BSD-2-Clause

**Comment by Johnnynator** [comment]
KDE Licensing is so damn annoing. Even one project can vary between e.g. `LGPL-2.0-only` and `LGPL-2.0-or-later` depending on which file you look at. And also a lot of nonstandard headers that are not using the default LGPL / GPL Header text

```
Source files that are part of a library with a public API which is part of the KDE Platform (kdelibs, kdepimlibs, kde-runtime and KDE Frameworks) must be licensed under:

    LGPL version 2.1, or version 3 or later versions approved by the membership of KDE e.V.

    They may also be licensed under one of the following terms if it helps with license compatibility:

    LGPL version 2.1 or later
    BSD license as listed below.
    Ensure that the BSD license does not contain the so called 'advertisement clause' or terms similar to Facebook's Additional Grant of Patent Rights.
    MIT license as listed below.
    X11 license as listed below.
```
Regarding this text there can't be any LGPL-2.0-only files but there still are. KDE Licensing is weird and not even following their own rules.

**Comment by maxice8** [comment]
needs to be rebased

**Pull request merged**